core/src/main/java/com/graphhopper/util/PointList.java

```diff
@@ -242,14 +242,23 @@ public void add(GHPoint point) {
     }
 
     public void add(PointList points) {
-        int newSize = size + points.getSize();
+        this.add(points, 0, points.size());
+    }
+
+    public void add(PointList points, int from, int to) {
```

Comment:
I added this method to have a better performance compared to first copy the PointList and then add it.

Reply:
You are right, might be the better solution. I did it like this, because at first I wanted to remove the first point of the PointList, which turned out to be the worse approach :). Once it was implemented doing the other war around worked easy. Should we switch to the `removeLast` approach?

Reply:
I'm fine with the current state